src/plugins/orangehrmAdminPlugin/test/fixtures/testcases/I18NLanguagesAPITestCase.yml

```diff
@@ -38,6 +38,39 @@ GetAll:
     meta:
       total: 3
 
+GetOne:
+  'with nonexistent language id':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    attributes:
+      id: 1000
+    exception:
+      class: \OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException
+      message: 'Record Not Found'
+
+  'with deleted language id':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    attributes:
+      id: 7
+    exception:
+      class: \OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException
+      message: 'Record Not Found'
```

Comment:
this test is the same as the previous since only 5 language records are loaded into test DB from the fixtures

we need to add a new parameter to getOne endpoint like getAll's `activeOnly`
https://github.com/orangehrm/orangehrm/blob/main/src/plugins/orangehrmAdminPlugin/Api/I18NLanguageAPI.php#L52-L57